RRS/customer/urls.py

```diff
@@ -10,8 +10,8 @@
         name  = 'index'
     ),
     url(
-        regex = r'^(\d+)/$',
+        regex = r'restaurant/(\d+)$',
         view = views.restaurant,
-        name="restaurant"
+        name="view-restaurant"
```

Comment:
> **RRS/customer/urls.py**
>  - 15: This is a matter of preference but I prefer the url `name` parameter to be the same as the static text part of the `regex` parameter to avoid confusion.

We can put the app name, and then add some verbs to complement the static url link. Something like `customer-view-restaurant`, for example. Would this be better?